### ISSUE TITLE
Some questions about repeating the results on SParC.

### ISSUE THREAD

**Issue by RandomTuringDuck** [ryanzhumich/editsql#16]
I want to reproduce the result of SParC on my server, and i only download the code and run the run.sh following the README. The model 'saved_42' is chosen to get the final result. 
![2020-03-11 22-10-57屏幕截图](https://user-images.githubusercontent.com/13444256/76425935-4a187980-63e5-11ea-93a7-f77d1af19d26.png)
But the final result is only 46.0, which is lower than the results in the paper. Although I repeat a lot of times, I get some similar results like 45.8 or 45.5. 
![2020-03-11 22-19-52屏幕截图](https://user-images.githubusercontent.com/13444256/76427380-2e15d780-63e7-11ea-8b28-31912bb2540c.png)
Maybe are the experimental environments  different? or you use some the different parameters, like learning rate or something.
I hope you can give me some suggestions.

**Comment by runzewang** [comment]
I run this code 5 times,  and the average question accuracy was 45.4% and the average interaction accuracy was 26.46%. 
If you reproduce the results,  please give me some suggestions. Thank you!

**Comment by huybery** [comment]
@RandomTuringDuck  Have you reproduced the results on Sparc ?

**Comment by Param-Raval** [comment]
When I train the model on SParC and run the testing on the saved model using test_sparc_editsql.sh for some reason it throws this error `Unexpected key(s) in state_dict: "utterance_attention_module.query_weights". `
Can you tell me what's the issue @ryanzhumich ?
